Portal/utils/SyncDocs/Program.cs

```diff
@@ -44,10 +45,9 @@
 
         // delete unnecessary files
         fileMappingService.CleanUpMappings();
-
+        if (string.IsNullOrWhiteSpace(deeplKey)) throw new InvalidOperationException($"Deepl Key is missing");
```

Comment:
💗

Reply:
lol - this prevents to delete all the docs when there is no valid key